### PR TITLE
Fix autoyast schema sle12sp1

### PR DESCRIPTION
**Pull request by imobach**
Port fixes from [bsc#805275](https://bugzilla.suse.com/show_bug.cgi?id=805275) also present in SLE 12.

### CODE REVIEW

**Comment by mvidner** [comment]
For the record, this is a port of #47 which was for SLE11.

File: Rakefile [hunk 1]
```diff
@@ -5,4 +5,5 @@ Yast::Tasks.submit_to :sle12sp1
 Yast::Tasks.configuration do |conf|
   #lets ignore license check for now
   conf.skip_license_check << /.*/
+  conf.obs_sr_project = "SUSE:SLE-12-SP1:Update"
```
Comment (mvidner) [inline Rakefile]: I don't understand, line 3 should be enough, shouldn't it?

```rb
 Yast::Tasks.submit_to :sle12sp1
```

Comment (imobach) [inline Rakefile]: @mvidner This is intended to be release as a maintenance update (it's not a P1 or a P2) and `sle12sp1` points to `SUSE:SLE-12-SP1:GA`. Also maybe we should update `yast-rake` as soon as SP1 is finally released.

Comment (mvidner) [inline Rakefile]: OK, so at some point ` Yast::Tasks.submit_to :sle12sp1` will be switched from GA to Update, and the explicit `conf.obs_sr_project` is to make it independent of that.
LGTM.

**Comment by ancorgs** [comment]
As commented during the review meeting, this should not be merged into SLE-12-SP1 branch. But into a new branch called `sle-12-sp1-after_release`.

See the thread titled "SLE12-SP1: Post-RC2 Behavior" in yast-internal mailing list

**Comment by imobach** [comment]
I've created a new PR for `sle-12-sp1-after_release`: https://github.com/yast/yast-kdump/pull/51

**Comment by imobach** [comment]
Thanks for the review and the comments.

**Pull request closed without merging**